CHANGELOG.md

```diff
@@ -1,29 +1,10 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
```

Comment:
you can keep this line here.  just put an extra space after this line and before the next heading